docs/sources/reference/commandline/cli.md

```diff
@@ -101,6 +101,7 @@ expect an integer, and they can only be specified once.
       --mtu=0                                Set the containers network MTU
       -p, --pidfile="/var/run/docker.pid"    Path to use for daemon PID file
       --registry-mirror=[]                   Preferred Docker registry mirror
+      -r, --root=""                          Set root user/uid remap option
```

Comment:
Sounds good to me; I'm fine with simply `--root`. @tibor: What do you think of @moxiegirl's suggestion on a "self-documenting" option like `--remap-root`?